generators/client-2/templates/_package.json

```diff
@@ -38,6 +39,7 @@
     "swagger-ui": "2.2.8",
     "ui-router-ng2": "1.0.0-beta.3",
     "ui-router-visualizer": "2.0.11",
+    "systemjs-plugin-css": "^0.1.29",
```

Comment:
or even better create a new branch from current angular2 and apply your changes there it would be much cleaner.